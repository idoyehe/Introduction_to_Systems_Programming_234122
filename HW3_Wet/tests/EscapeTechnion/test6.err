Error: The email entered does not belong to any company in the system
Error: The parameter sent to the function is invalid
Error: The room ID does not belong to any room in the faculty
Error: The room ID does not belong to any room in the faculty
Error: The email entered does not belong to any client in the system
Error: There is no room available to recommend
