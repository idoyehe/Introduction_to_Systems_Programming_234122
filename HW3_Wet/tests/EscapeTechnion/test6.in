report day
report best

company remove bio4@gmail.com
room remove 9 -33
room remove 9 33
room remove 1 33
escaper remove escaper2@gmail.com

escaper add escaper1@gmail.com 3 5

escaper recommend escaper1@gmail.com 12

report day
report best

### end ###