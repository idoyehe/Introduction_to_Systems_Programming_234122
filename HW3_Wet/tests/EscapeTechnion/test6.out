Day #0: 0 events in total
That is all for day #0!

The system has 18 faculties and in 1 days, they have made a revenue of 0
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 0 so far!
One of the best faculties - Mechanical Engineering, has made a revenue of 0 so far!
One of the best faculties - Electrical Engineering, has made a revenue of 0 so far!

Day #1: 0 events in total
That is all for day #1!

The system has 18 faculties and in 2 days, they have made a revenue of 0
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 0 so far!
One of the best faculties - Mechanical Engineering, has made a revenue of 0 so far!
One of the best faculties - Electrical Engineering, has made a revenue of 0 so far!

